root/statistics/timeline.tt

```diff
@@ -20,10 +21,11 @@
         </table>
         <div id="overview" data-bind="flot: 'overview'"></div>
     </div>
+    [%- USE Translation('mb_server') -%]
 [%~ END ~%]
 
 [%~ WRAPPER "statistics/layout.tt" title=l("Timeline graph") sidebar=sidebar page='timeline' ~%]
-[%~ PROCESS "statistics/macros-header.tt" ~%]
+[%- USE Translation('statistics') -%]
```

Comment:
The `StatisticsCSS` gets included via root/statistics/layout.tt AFAICT (just mentioning it because I wasn't sure).